tools/biolatency.py

```diff
@@ -139,9 +143,9 @@
         b.attach_kprobe(event="blk_start_request", fn_name="trace_req_start")
     b.attach_kprobe(event="blk_mq_start_request", fn_name="trace_req_start")
 b.attach_kprobe(event="blk_account_io_done",
-    fn_name="trace_req_done")
+                fn_name="trace_req_done")
 
-print("Tracing block device I/O... Hit Ctrl-C to end.")
+# print("Tracing block device I/O... Hit Ctrl-C to end.")
```

Comment:
You do not want to unconditionally comment out the above banner. You can guard it for non-jason output.

Reply:
I have put it within an `if` block now. Will not print banner only when the json output is being written to a file.